src/Motion.js

```diff
@@ -42,22 +50,21 @@ const Motion = createReactClass({
       lastIdealStyle: currentStyle,
       lastIdealVelocity: currentVelocity,
     };
-  },
+  }
 
-  wasAnimating: false,
-  animationID: (null: ?number),
-  prevTime: 0,
-  accumulatedTime: 0,
+  animationID: ?number = null;
```

Comment:
Yes :) I was just testing with one property to see if different result comes out compare to my local test